code/modules/mob/living/silicon/ai/vox_sounds.dm

```diff
@@ -912,7 +912,6 @@ GLOBAL_LIST_INIT(vox_sounds, list("abduction" = 'sound/vox_fem/abduction.ogg',
 "victor" = 'sound/vox_fem/victor.ogg',
 "violated" = 'sound/vox_fem/violated.ogg',
 "violation" = 'sound/vox_fem/violation.ogg',
-"virologist" = 'sound/vox_fem/virologist.ogg',
```

Comment:
actually remove this file or put it back

Reply:
Reverted this removal